go-controller/pkg/ovn/service.go

```diff
@@ -205,14 +327,15 @@ func (ovn *Controller) deleteService(service *kapi.Service) {
 					protocol, err)
 				break
 			}
-
-			err = ovn.createLoadBalancerVIP(loadBalancer,
-				service.Spec.ClusterIP, svcPort.Port, ips, targetPort)
-			if err != nil {
-				klog.Errorf("Error in deleting load balancer for service "+
-					"%s %+v", util.JoinHostPortInt32(service.Name, port), err)
-			}
-			ovn.handleExternalIPs(service, svcPort, ips, targetPort)
+			vip := util.JoinHostPortInt32(service.Spec.ClusterIP, svcPort.Port)
+			ovn.deleteLoadBalancerVIP(loadBalancer, vip)
+			ovn.handleExternalIPs(service, svcPort, ips, targetPort, true)
 		}
 	}
 }
+
+// svcQualifiesForReject determines if a service should have a reject ACL on it when it has no endpoints
```

Comment:
@trozet this comment should say something about idling.  Eg that Service idling and reject ACL are opposite sides of the coin when the service has no endpoints.

Reply:
OK will add more detail here.